packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -0,0 +1,291 @@
+/* eslint-disable global-require, import/no-dynamic-require, react/no-unused-prop-types, react/no-danger */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import ResponsiveElement from 'terra-responsive-element';
+
+import './DemographicsBanner.scss';
+
+const propTypes = {
+  /**
+   * Additional Details to display in the banner.
+   */
+  additionalDetails: PropTypes.node,
+  /**
+   * Determines whether to truncate the application content on small screen sizes
+   */
+  additionalDetailsTruncated: PropTypes.bool,
```

Comment:
We actually don't need this variable anymore.

I talked to UX, and since we don't have a good way of controlling truncation, we aren't going to do anything with it. It'll be up to the consumer to determine the correct truncation styles and components for their specific content.